charts/images.yaml

```diff
@@ -3,22 +3,22 @@ images:
   resourceId:
     name: 'etcdbrctl'
   sourceRepository: github.com/gardener/etcd-backup-restore
-  repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
+  repository: europe-docker.pkg.dev/gardener-project/releases/gardener/etcdbrctl
```

Comment:
@ccwienk Any specific reason to use `releases` and not `public` registry here? Because the PR uses `public` registry in other files, such as `api/v1alpha1/types_etcd_test.go`, `test/utils/etcd.go` and `test/utils/etcdcopybackupstask.go`.

Reply:
two reasons:
- assuming helmcharts are used to install release-versions, I preferred configuring `releases` over `public`, because the latter will also show snapshot-versions (which are also subject to automatic cleanup)
- image-references from images.yaml are used in conjunction with those configured in .ci/pipeline_definitions to add mapping-labels (done by "component-cli"). I wanted to avoid surprises and preferred to keep those image-references in sync

Reply:
Helm charts could also be used to install snapshots, especially when testing. Wouldn't it then be better to keep the repository as `public`, so that it can use either release or snapshot images?

Reply:
@shreyas-s-rao : okay, changed to `public`.